addons/metallb/00_metallb-namespace.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
Your files are missing copyright header, see for example:https://github.com/kubermatic/kubermatic/blob/master/addons/cilium/cilium.yaml#L1-L13

Reply:
Sure.